include/ostatus.php

```diff
@@ -1050,6 +1138,7 @@ private function completion($conversation_url, $uid, $item = array(), $self = ""
 				continue;
 			}
 
+			/// @TODO One statment is okay (until if() )
```

Comment:
No, it is commonly not a good practice, to have tons of nested curly braces around as it makes the code more complicated (to read/review by human not to parse by program).